packages/ceramic-http-client/package.json

```diff
@@ -27,19 +27,19 @@
     "clean": "rm -rf ./lib"
   },
   "dependencies": {
+    "@types/node-fetch": "^2.5.5",
```

Comment:
This should be in dev-deps.

Reply:
We use `node-fetch` in production code for this package, so the types are needed as well

Reply:
hm, not sure I follow. The types should only be needed when compiling?